src/SAWScript/Interpreter.hs

```diff
@@ -3180,6 +3180,15 @@ primitives = Map.fromList
     , "the live variables in the loop evolve as the loop computes."
     ]
 
+  , prim "llvm_verify_x86_with_invariant"
+    "LLVMModule -> String -> String -> [(String, Int)] -> Bool -> (String, Int, Term) -> LLVMSetup () -> ProofScript () -> TopLevel LLVMSpec"
+    (pureVal llvm_verify_x86_with_invariant)
+    Experimental
+    [ "An experimental variant of 'llvm_verify_x86'. This variant can prove some properties"
```

Comment:
Heh, perhaps it's a good idea to use a word other than "variant" here to avoid confusion with "invariant".